src/MizarData/MizarPairedDataTest.cpp

```diff
@@ -253,33 +264,40 @@ TEST_F(MizarPairedDataTest, ForeachCallstackIsCorrect) {
   EXPECT_THAT(actual_ids_fed_to_action, UnorderedElementsAre(kAnotherCompleteCallstackIds));
 }
 
+const auto kExpectedInvocationTimes = {Times(kSamplingPeriod, uint64_t{2}),
```

Comment:
Nit: It's needed to wrap the 2 in a uint64_t?

Reply:
Yeah, it's a a limitation of typedef arithmetics. We've declared multiply-ability with uint64_t and we have to supply it.